configs/webserver/halfstack.conf

```
@@ -39,6 +39,7 @@ jwt.secret = "7<:~[X,^Z1XM!*,Pe:PHR!bv,H~Q#l177<7gf_XHD6.<*<.t<[o|V5W(=0x:jTh-"
 
 [ui]
 brand = "Lablup Cloud"
+menu_blocklist = "pipeline"
 #menu_blocklist = "statistics,pipeline"
```

```suggestion
menu_blocklist = "pipeline"
```

It seems that the comments in the `menu_blocklist` were not removed by mistake.